docs/developer/developer-manual.html

```diff
@@ -471,7 +472,34 @@ <h2 id="Making_a_Checker_Framework_release">Making a Checker Framework release</
 <!-- This relative URL works from a clone, but not from the web. -->
 <a href="release/README-release-process.html">local version (link works from a clone)</a> or
 <a href="https://rawgit.com/typetools/checker-framework/master/docs/developer/release/README-release-process.html">web version (from previous release)</a>.
+</p>
+
 
+<h2 id="Build_historical">Building a historical version of the Checker Framework</h2>
+
+<p>
+The use of four different repositories, as explained in
+"<a href="#Related_repositories">Related repositories</a>" above, means that you
+cannot check out an old version of the Checker Framework and build it
+with <code>./gradlew assemble</code>.  By default, the Checker Framework build
+uses the latest version of the other three repositories, which are rarely
+compatible with an old version of the Checker Framework.  One symptom of the
+problem is a "Can't find stubparser jar" error message.
+</p>
+
+<p>
+To build an old version of the Checker Framework, you need to use old versions
+of the other repositories as well.  (Clone them into siblings of
```

Comment:
Recently, any stubparser change that breaks building the Checker Framework, should have a new version number.  So stubparser should be checked out to the same commit where the version number was changed to what it is in checker-framework/build.gradle.